Alexa.NET.Management/Api/CustomApiEndpoint.cs

```diff
@@ -1,14 +1,17 @@
 ﻿using Newtonsoft.Json;
 using Newtonsoft.Json.Converters;
+using System;
+using System.Collections.Generic;
+using System.Text;
 
 namespace Alexa.NET.Management.Api
 {
-    public class CustomApiRegionEndpoint
+    public class CustomApiEndpoint
     {
-        [JsonProperty("sslCertificateType"), JsonConverter(typeof(StringEnumConverter))]
-        public SslCertificateType SslCertificateType { get; set; }
-
         [JsonProperty("uri")]
         public string Uri { get; set; }
+
+        [JsonProperty("sslCertificateType"), JsonConverter(typeof(StringEnumConverter), NullValueHandling.Ignore)]
```

Comment:
NullValueHandling needs to be part of the JsonPropertyAttribute instead of the JsonConverter attribute